app/models/megaphone/episode.rb

```diff
@@ -98,8 +98,7 @@ def self.attributes_from_episode(e, feed)
         background_image_file_url: e.ready_image&.href,
         episode_type: e.itunes_type,
         episode_number: e.episode_number,
-        season_number: e.season_number,
-        ad_free: e.categories.include?("adfree")
```

Comment:
no point in setting this - I think it is actually informational? like, if it is adFree (no ads setup), then this is true, but setting it is a no-op